includes/elements/container.php

```diff
@@ -257,7 +267,27 @@ public function before_render() {
 	 * @return void
 	 */
 	public function after_render() {
-		?></<?php $this->print_html_tag(); ?>><?php
+		$settings = $this->get_settings_for_display();
+		if ( $this->is_boxed_container( $settings ) ) {
+			?></div><?php
+		}
+		?>
+		</<?php $this->print_html_tag(); ?>><?php
+	}
+
+	/**
+	 * Is Boxed Container.
+	 *
+	 * @param array $settings
+	 *
+	 * @return boolean
+	 */
+	private function is_boxed_container( $settings ) {
```

Comment:
you can use types for the params and the return, no need to document

Reply:
@TzviRabinovitch Done